tests/dotnet/UnitTests/BundleStructureTest.cs

```diff
@@ -279,8 +279,10 @@ internal static void CheckAppBundleContents (ApplePlatform platform, IEnumerable
 			}
 
 			// misc other files not directly related to the test itself
-			if (!isCoreCLR)
+			if (!isCoreCLR) {
 				expectedFiles.Add (Path.Combine (assemblyDirectory, "icudt.dat"));
+				expectedFiles.Add (Path.Combine (assemblyDirectory, "icudt_hybrid.dat"));
```

Comment:
A test is failing because of this change, because the test is now expecting icudt_hybrid.dat to be produced by the build - but it's disabled by default, so that doesn't hapepn.

Reply:
Thanks! Removed this test case change.